src/main/java/javaClasses/homework_4/arevik_arakelyan/B/Staff.java

```diff
@@ -0,0 +1,31 @@
+package javaClasses.homework_4.arevik_arakelyan.B;
+
+public abstract class Staff {
+    protected String names;
+    protected String surnames;
+    protected  final static   String COMPANY = "My Company";
+
```

Comment:
Please provide an implementation of constructor so you can create objects with providing all necessary fields.

Reply:
done